SMTP workflow improvements (#30)
forem/admin-docs: docs/advanced-customization/config/email-server-settings.md

---

> description: Configure your Forem's Email Server Settings.

Author: We've renamed SMTP Settings to a more human readable "Email Server Settings"

position: 5
---

# Email Server Settings (SMTP)

## Forem Cloud Clients

> As a Forem Cloud client, we provide an email server managed by the Forem team. All settings are managed by us and the from and reply email addresses for all emails are set as noreply@forem.com.

Author: We've added the ability for Forem Cloud Clients to override their Sendgrid settings and use custom SMTP settings if they wish to.

> As a Forem Cloud client, we provide an email server managed by the Forem team. All settings are managed by us and the from and reply email addresses for all emails are set as noreply@forem.com.

Author: ```suggestion
As a Forem Cloud client, you are provided with an email server managed by the Forem team.
```

> As a Forem Cloud client, we provide an email server managed by the Forem team. All settings are managed by us and the from and reply email addresses for all emails are set as noreply@forem.com.

Author: Updated, thank you Ella ❤️

If you need to specify a HELO domain, you can do it here. If not, leave this field blank.

### Password
### Port

Password specified for your SMTP service (not always the password for accessing your account). Sendgrid users will need to generate an API key instead, and [use the full API key as the password](https://docs.sendgrid.com/for-developers/sending-email/upgrade-your-authentication-method-to-api-keys#upgrade-to-api-keys-for-your-smtp-integration). [Sendinblue users will need to request an SMTP password from Support before being able to use SMTP](https://help.sendinblue.com/hc/en-us/articles/209463245).
The port that your mail server runs on. This could be 25 or 587 for unencrypted/TLS email, or 465 for SSL-encrypted email, so it's best to confirm these settings with your SMTP service provider.

> ### Domain

Author: We've added a from and reply to email address
